src/Features/Core/Portable/Completion/CommonCompletionProvider.cs

```diff
@@ -92,7 +93,7 @@ public override async Task<CompletionChange> GetChangeAsync(Document document, C
             => GetTextChangeAsync(selectedItem, ch, cancellationToken);
 
         protected virtual Task<TextChange?> GetTextChangeAsync(CompletionItem selectedItem, char? ch, CancellationToken cancellationToken)
-            => Task.FromResult<TextChange?>(null);
```

Comment:
>Task.FromResult<TextChange?>(null); [](start = 15, length = 35)

Isn't null optimized in the Task class?

Reply:
No. `Task.FromResult<T>` always returns a new instance.